webapp/src/js/enketo/bootstrap-datepicker.bm.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-extra-semi */
```

Comment:
How was this working before? Did we actually run JSHint on it prior (ie how did the double quotes not throw issues?).

It might be that we were intentionally not linting it, and may want to continue

Reply:
jshint igonred this file, I will apply the same